[ALL TESTS] Update (#5877)
mlflow/mlflow: CHANGELOG.md

# CHANGELOG

## 1.26.0 (2022-05-16)

MLflow 1.26.0 includes several major features and improvements:

Features:

- [CLI] Add endpoint naming and options configuration to the deployment CLI (#5731, @trangevi)
- [Build,Doc] Add development environment setup script for Linux and MacOS x86 Operating Systems (#5717, @BenWilson2)

> - [Tracking] Update `mlflow.set_tracking_uri` to add support for paths defined as `pathlib.Path` in addition to existing `str` path declarations (#5820) (#5824, @cacharle)

Reviewer: ```suggestion
- [Tracking] Update `mlflow.set_tracking_uri` to add support for paths defined as `pathlib.Path` in addition to existing `str` path declarations (#5824, @cacharle)
```

> - [Scoring] Add custom timeout override option to the scoring server CLI to support high latency models #5381 (#5663, @sniafas)

Reviewer: ```suggestion
- [Scoring] Add custom timeout override option to the scoring server CLI to support high latency models(#5663, @sniafas)
```

- [UI] Add sticky header to experiment run list table to support column name visibility when scrolling beyond page fold (#5818, @hubertzub-db)
- [Artifacts] Add GCS support for MLflow garbage collection (#5811, @aditya-iyengar-rtl-de)
- [Evaluate] Add `pos_label` argument for `eval_and_log_metrics` API to support accurate binary classifier evaluation metrics (#5807, @yxiong)
- [UI] Add fields for latest, minimum and maximum metric values on metric display page (#5574, @adamreeve)
- [Models] Add support for `input_example` and `signature` logging for pyspark ml flavor when using autologging (#5719, @bali0019)
- [Models] Add `virtualenv` environment manager support for `mlflow models docker-build` CLI (#5728, @harupy)
- [Models] Add support for wildcard module matching in log_model_allowlist for PySpark models (#5723, @serena-ruan)

> - [Projects] Add `virtualenv` environment manager support for MLflow projects (#5631, @harupy)

Reviewer: ```suggestion
- [Projects] Add `virtualenv` environment manager support for MLflow Models (#5380, @harupy)
```

Let's move #5380 here.

> - [Projects] Add `virtualenv` environment manager support for MLflow projects (#5631, @harupy)

Reviewer: ```suggestion
- [Projects] Add `virtualenv` environment manager support for MLflow projects (#5631, @harupy)
- [Models] Add `virtualenv` environment manager support for MLflow Models (#5380, @harupy)
```

Let's move #5380 here.

- [Models] Add `virtualenv` support for `mlflow.pyfunc.spark_udf` (#5676, @WeichenXu123)
- [Models] Add support for `input_example` and `signature` logging for `tensorflow` flavor when using autologging (#5510, @bali0019)
- [Server-infra] Add JSON Schema Type Validation to enable raising 400 errors on malformed requests to REST API endpoints (#5458, @mrkaye97)
- [Scoring] Introduce abstract `endpoint` interface for mlflow deployments (#5378, @trangevi)

> - [UI] Add 'end time' and 'run duration' fields to Compare Runs view (#3378, @RealArpanBhattacharya)

Reviewer: ```suggestion
- [UI] Add `End Time` and `Duration` fields to run comparison page (#3378, @RealArpanBhattacharya)
```
